port/unix/omrsyslog.c

```diff
@@ -153,6 +154,11 @@ get_conv_txt_num_bytes(struct OMRPortLibrary *portLibrary, const char *message)
 static void
 syslog_write(struct OMRPortLibrary *portLibrary, const char *buf, int priority)
 {
+#if defined(OMRZTPF)
+	sprintf(buf, "%s", buf);
```

Comment:
This looks suspicious: `buf` should be considered read-only and shouldn't be the target of `sprintf` (even if it only writes what's already there).